IBX-7489: `Change the password` section added (#292)
ibexa/documentation-user: docs/getting_started/get_started.md

--------|-----------|
|Active dashboard|Sets which dashboard is displayed after you log in.|

### Change the password

> You can change your user [password]([[= developer_doc =]]/users/passwords/) at any time.

Reviewer: ```suggestion
You can change your user [password]([[= developer_doc =]]/users/passwords/) at any time.
```

I would add a disclaimer here: "If the administrator granted you the right permissions, ..."

Author: It's explained in the "Passwords" page mentioned in link.

To do it, first, access your user profile, and go to **Account settings** tab.
Then, click **Change password**.

![Change password](img/change_password.png "Change password")

> Fill out all the required fields and click **Save and close** to save changes.

Reviewer 2: ```suggestion
Fill in all the required fields and click **Save and close** to save changes.
```

> Click **Discard** to remove your changes and return to the previous screen.

Reviewer 2: ```suggestion
Click **Discard** to reject changes and return to the previous screen.
```
